cpp/src/bindings.cpp

```diff
@@ -23,6 +22,55 @@ void bindArray(py::module_& m, const std::string& className) {
             }, py::keep_alive<0, 1>());
 }
 
+array* createArray(py::list columnsValues, py::list columnsNames, py::list columnsTypes, int columnSize, int columnsNumber) {
+    array* dataframe = new array();
+    dataframe->ptr = new series[columnsNumber];
+    series* columns = new series[columnsNumber];
+    for (int indice = 0 ; indice < columnsNumber ; indice ++ ) {
+        series* column = columns + indice;
+        py::str name = (py::str) columnsNames[indice];
+        column->name = strdup((char*) const_cast<char*>(((std::string) name).c_str()));
+        column->index = 0;
```

Comment:
actually here we could set it to true/false according to the index list, that way on the java side we will know which series is an index without a separate list.